pkg/synopsysctl/cmd_serve_UI.go

```diff
@@ -519,10 +513,13 @@ func convertPolarisUIResponseToPolarisObject(polarisUIRequestConfig PolarisUIReq
 	}
 	polarisObj.Version = polarisUIRequestConfig.Version
 	polarisObj.EnvironmentDNS = polarisUIRequestConfig.EnvironmentDNS
-	polarisObj.ImagePullSecrets = polarisUIRequestConfig.ImagePullSecrets
+	data, err := util.ReadFileData(polarisUIRequestConfig.ImagePullSecrets)
```

Comment:
`ImagePullSecrets` this might get confusing since in this case this is really `GCPServiceAccount` and there is another possibility for `ImagePullSecrets`.  Also, generally, is there a way to possibly reuse the same struct from `synopsysctl`, so we don't have mismatches in maintenance in the future also.

Reply:
I changed the name. I'll look into the struct usage in a different PR.